documentation/src/main/asciidoc/user_guide/counters.adoc

```diff
@@ -157,46 +162,56 @@ EmbeddedCacheManager manager = ...;
 CounterManager counterManager = EmbeddedCounterManagerFactory.asCounterManager(manager);
 ----
 
+==== Remove a counter via CounterManager
+
+WARNING: use with caution.
+
+There is a difference between remove a counter via the `Strong/WeakCounter` interfaces and the `CounterManager`.
+The `CounterManager.remove(String)` removes the counter value from the cluster and removes all the listeners registered
```

Comment:
Please describe what a regular remove does when you already mention that there's a difference.

Reply:
done!